packages/rpc-provider/src/ws/polyfill.spec.ts

```diff
@@ -2,25 +2,18 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-describe('ws/polyfill', () => {
-  let origWs;
+import { Global } from '@polkadot/rpc-provider/mock/types';
```

Comment:
I also could need a second pair of eyes on this file / tests.

Reply:
Agreed, they are not testing what they should be. Have a look at the HTTP one above

Reply:
Yeah, that also doesn't really test what it's supposed to. Both tests are basically testing the same thing